adoc/TRD-wordpress-on-lamp-on-sles.adoc

```diff
@@ -574,7 +574,7 @@ success
 systemctl start apache2
 ----
 
-. Open a browser with your server IP or host name at http://`hostname`. If the installation was successful, you should be presented with the WordPress installation page. 
+. Open a browser with your server IP or hostname at http://`hostname`. If the installation was successful, you should be presented with the WordPress installation page. 
```

Comment:
SUSE Style Guide says "host name" (no matter if in the command then you have `hostname`)

```suggestion
. Open a browser with your server IP or host name at http://`hostname`. 
```